packages/devtools/lib/src/timeline/cpu_profile_protocol.dart

```diff
@@ -105,6 +117,13 @@ class CpuProfileData {
       parent.addChild(stackFrame);
     }
   }
+
+  void _setLeafCounts() {
```

Comment:
https://docs.microsoft.com/en-us/visualstudio/profiling/understanding-sampling-data-values?view=vs-2019#exclusive-samples
instead of leaf counts lets use exclusiveCount.

Reply:
leafCount --> exclusiveSampleCount